scripts/ci/pre_commit/pre_commit_check_pre_commit_hooks.py

```diff
@@ -25,7 +25,7 @@
 import sys
 from pathlib import Path
 
-sys.path.insert(0, str(Path(__file__).parent.absolute()))  # make sure common_precommit_utils is imported
+sys.path.insert(0, str(Path(__file__).parent.resolve()))  # make sure common_precommit_utils is imported
```

Comment:
Same here, `sys.path` accepts a relative path just fine and I don’t think we need to convert. (There are several places using this pattern and this should apply to them all.)

Reply:
Same as above. It does not hurt. Call it being overcautiious but I do not want to exercise any extra mental power to remember where I can use relative path and where it is not safe. Using absolute path is always safe.